src/content/docs/start/prerequisites.mdx

```diff
@@ -202,6 +181,51 @@ Install WebView2 by visiting the [WebView2 Runtime download section](https://dev
 
 Next: [Install Rust](#rust)
 
+### Nix/NixOS
+
+Using a [`shell.nix`](https://nix.dev/tutorials/first-steps/declarative-shell):
+
+```nix
+let
+  pkgs = import <nixpkgs> {
+    config = { };
+    overlays = [ ];
+  };
+in
+
+pkgs.mkShell {
+  nativeBuildInputs =
+    with pkgs;
+    [
+      rustc
+      cargo
+      # Uncomment if `@tauri-apps/cli` doesn't work on your system
+      # cargo-tauri
+      pkg-config
+    ]
+    ++ lib.optionalAttrs stdenv.hostPlatform.isLinux [ wrapGAppsHook4 ];
+
+  buildInputs =
+    with pkgs;
+    [ openssl ]
```

Comment:
@getchoo for some reason `libiconv` is required maybe for plugins?

Reply:
What plugins specifically? It'd be nice to note it

I also imagine that would be a macOS exclusive dependency; libiconv is hardly ever used on Linux

Reply:
I would have to test to see which plugins require it.